### DISCUSSION TITLE
How to remove "Made with Chatwoot" from the help center?

### DISCUSSION THREAD

**Discussion by flaviomoceri** [chatwoot/chatwoot#8573] [Self hosted]
I'm hosting Chatwoot using docker, I created a help center and the words "Made with Chatwoot" appear in the footer, is there a way to remove it?
![Screenshot 2023-12-16 082519](https://github.com/chatwoot/chatwoot/assets/63161412/aae465e0-fab8-451c-8f01-f3006afedd93)

**Comment by palmeirarocha** [comment]
Paying

**Reply by lpsoliveira** [reply]
Yes Paying...
